lib/js/dom.nim

```diff
@@ -985,6 +985,16 @@ type
     once*: bool
     passive*: bool
 
+since (1, 3):
+  type DOMParser* = ref object  ## \
```

Comment:
Bikeshedding: Should this be `DomParser` (and `newDomParser`)? Quoting [NEP-1](https://nim-lang.org/docs/nep1.html#introduction-naming-conventions):

> In the age of HTTP, HTML, FTP, TCP, IP, UTF, WWW it is foolish to pretend these are somewhat special words requiring all uppercase. Instead treat them as what they are: Real words. So it's parseUrl rather than parseURL, checkHttpHeader instead of checkHTTPHeader etc.

Don't know if DOM counts on the same level as HTTP and HTML but I don't really know if the "level" matters either.

Reply:
Yeah, it's DomParser and newDomParser indeed.